src/core/pdf_manager.js

```diff
@@ -15,152 +15,149 @@
-    },
-  };
+  terminate() {
+    unreachable('Abstract method `terminate` called');
+  }
+}
 
-  return BasePdfManager;
-})();
+class LocalPdfManager extends BasePdfManager {
+  constructor(docId, data, password, evaluatorOptions, docBaseUrl) {
+    super();
 
-var LocalPdfManager = (function LocalPdfManagerClosure() {
-  function LocalPdfManager(docId, data, password, evaluatorOptions,
-                           docBaseUrl) {
     this._docId = docId;
     this._password = password;
     this._docBaseUrl = docBaseUrl;
     this.evaluatorOptions = evaluatorOptions;
-    var stream = new Stream(data);
+
+    const stream = new Stream(data);
     this.pdfDocument = new PDFDocument(this, stream);
     this._loadedStreamCapability = createPromiseCapability();
     this._loadedStreamCapability.resolve(stream);
   }
 
-  Util.inherit(LocalPdfManager, BasePdfManager, {
-    ensure: function LocalPdfManager_ensure(obj, prop, args) {
-      return new Promise(function (resolve, reject) {
-        try {
-          var value = obj[prop];
-          var result;
-          if (typeof value === 'function') {
-            result = value.apply(obj, args);
-          } else {
-            result = value;
-          }
-          resolve(result);
-        } catch (e) {
-          reject(e);
+  ensure(obj, prop, args) {
+    return new Promise(function(resolve, reject) {
+      try {
+        const value = obj[prop];
+        let result;
+        if (typeof value === 'function') {
+          result = value.apply(obj, args);
+        } else {
+          result = value;
         }
-      });
-    },
+        resolve(result);
+      } catch (e) {
+        reject(e);
+      }
+    });
+  }
 
-    requestRange: function LocalPdfManager_requestRange(begin, end) {
-      return Promise.resolve();
-    },
+  requestRange(begin, end) {
+    return Promise.resolve();
+  }
 
-    requestLoadedStream: function LocalPdfManager_requestLoadedStream() {
-      return;
-    },
+  requestLoadedStream() {
+    return;
+  }
 
-    onLoadedStream: function LocalPdfManager_onLoadedStream() {
-      return this._loadedStreamCapability.promise;
-    },
+  onLoadedStream() {
+    return this._loadedStreamCapability.promise;
+  }
 
-    terminate: function LocalPdfManager_terminate() {
-      return;
-    },
-  });
+  terminate() {
+    return;
+  }
+}
 
-  return LocalPdfManager;
-})();
+class NetworkPdfManager extends BasePdfManager {
+  constructor(docId, pdfNetworkStream, args, evaluatorOptions, docBaseUrl) {
+    super();
 
-var NetworkPdfManager = (function NetworkPdfManagerClosure() {
-  function NetworkPdfManager(docId, pdfNetworkStream, args, evaluatorOptions,
-                             docBaseUrl) {
     this._docId = docId;
     this._password = args.password;
     this._docBaseUrl = docBaseUrl;
     this.msgHandler = args.msgHandler;
     this.evaluatorOptions = evaluatorOptions;
 
-    var params = {
+    const params = {
```

Comment:
You can probably just remove this temporary object, by doing e.g.
```javascript
this.streamManager = new ChunkedStreamManager(pdfNetworkStream, {
  msgHandler: args.msgHandler,
  url: args.url,
  length: args.length,
  disableAutoFetch: args.disableAutoFetch,
  rangeChunkSize: args.rangeChunkSize,
});
```

Reply:
Done in the new commit.